proposals/pattern-const-expressions.md

```diff
@@ -84,12 +107,13 @@ None.
 ## Alternatives
 [alternatives]: #alternatives
 
-Currently, equality and comparison operators can be used to compare against other literals, including nullability of the objects (e.g. `x != null`, or `y == 4 || y < 3`). However, type checking cannot be currently peformed at compile time and assigned to a constant.
+Currently, equality and comparison operators can be used to compare against other literals, including nullability of the objects (e.g. `x != null`, or `y == 4 || y < 3`).
 
 ## Unresolved questions
 [unresolved]: #unresolved-questions
 
 - [ ] Requires LDM review
+- [ ] Should we introduce a new error for non-constant subpatterns in order to isolate the root cause of the inability to consider the expression constant?
```

Comment:
This isn't a language concern. The compiler is free to make most-specific errors if it can and it would be helpful. It doesn't need to be specified.